### DISCUSSION TITLE
No "results"?

### DISCUSSION THREAD

**Discussion by degenaro** [kubescape/kubescape#1174] [Q&A]
I run kubescape against bookinfo.yml and all is well. Data produced comprises "results" and "summaryDetails" and more. Nice.

But I run kubescape against a collection of bitnami yml and get "summaryDetails" and more, but no "results". How come???

**Comment by matthyx** [comment]
Hi @degenaro can you please send us the exact YAMLs used in both cases?

**Comment by degenaro** [comment]
Hi @matthyx . This (failing) yaml was created by `helm search repo bitnami` and removing those that are DEPRECATED.

[ball.yaml.gz](https://github.com/kubescape/kubescape/files/11121802/ball.yaml.gz)

**Comment by matthyx** [comment]
Hi @degenaro I know the answer: in fact if you scan a file or a dir, there is no result sent because we would not know where to put them in the cloud GUI...
So in order to have results submitted, you should put your file inside a git repository and scan it.
I hope it helps.
